Orange/tests/test_preprocess.py

```diff
@@ -52,3 +54,24 @@ def test_nothing_to_remove(self):
         data = Orange.data.Table("iris")
         d = Orange.preprocess.preprocess.RemoveConstant(data)
         self.assertEqual(len(d.domain.attributes), 4)
+
+
+class TestRemoveNanClass(unittest.TestCase):
+    def test_remove_nan_classes(self):
+        table = Table("imports-85")
+        self.assertTrue(np.isnan(table.Y).any())
+        table = RemoveNaNClasses(table)
+        self.assertTrue(not np.isnan(table.Y).any())
```

Comment:
I'm just curious: is this the same as `self.assertFalse(np.isnan(table.Y).any())` or am I missing something?